src/components/Stakeholders/Table.js

```diff
@@ -0,0 +1,95 @@
+import React, { useContext } from 'react';
+import StakeholderDetail from './StakeholderDetail';
+import {
+  Paper,
+  TableContainer,
+  Table,
+  TableHead,
+  TableRow,
+  TableCell,
+  TableBody,
+  TableSortLabel,
+  TablePagination,
+} from '@material-ui/core';
+import { StakeholdersContext } from '../../context/StakeholdersContext';
+
+function StakeholderTable() {
+  const {
+    stakeholders,
+    columns,
+    page,
+    rowsPerPage,
+    orderBy,
+    order,
+    setPage,
+    setRowsPerPage,
+    sort,
+  } = useContext(StakeholdersContext);
+
+  const handleRowsPerPageChange = (e) => {
+    setRowsPerPage(e.target.value);
+  };
+
+  const handlePageChange = (e, page) => {
+    setPage(page);
+  };
+
+  const handleSort = (col, order) => {
+    console.log('sort col', { col, order });
+    sort({ col, order });
+  };
+
+  return (
+    <>
+      <TableContainer component={Paper}>
+        <Table aria-label="collapsible table">
+          <TableHead>
+            <TableRow>
+              {columns.map((col) => (
+                <TableCell
+                  key={col.value}
+                  onClick={() => handleSort(col.value, !order)}
+                >
+                  <TableSortLabel
+                    active={col.value === !!orderBy}
+                    direction={order ? 'asc' : 'desc'}
+                  >
+                    {col.label}
+                  </TableSortLabel>
+                </TableCell>
+              ))}
+            </TableRow>
+          </TableHead>
+          <TableBody>
+            {stakeholders.map((stakeholder) => (
+              <React.Fragment key={stakeholder.id}>
+                {/* Main stakeholder */}
+                <StakeholderDetail row={stakeholder} columns={columns} />
+                {stakeholder.children &&
+                  stakeholder.children.map((child) => (
+                    <StakeholderDetail
+                      key={child.id}
+                      row={child}
+                      columns={columns}
+                      child
+                    />
+                  ))}
+              </React.Fragment>
+            ))}
+          </TableBody>
+        </Table>
+        <TablePagination
+          component="div"
+          rowsPerPage={rowsPerPage}
+          rowsPerPageOptions={[1, 2, 5]}
```

Comment:
This doesn't seem right - is it just for testing?

Reply:
It is the count of the main (logged in) stakeholder rows, not including their children, but you're right I'll revisit